src/components/Navbar.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'; 
+import AppLogo from "./images/navbar-logo.png";
+
+
+function Navbar() {
+    return (
+        <nav className="app-header">
```

Comment:
still className is not in camel case.
eg : change "app-header" to "appHeader", Update all classNames accordingly